GSA-TTS/FAC
# Tweaking audit user management instructions

Addresses https://github.com/GSA-TTS/FAC/issues/3409

In this PR:
- Tweaking audit user management instructions to match [this](https://github.com/GSA-TTS/FAC/issues/3409#issuecomment-2408076401) and [this](https://github.com/GSA-TTS/FAC/issues/3409#issuecomment-2408079438)

Testing:
- Create an audit and go to the user management page. Instructions should look like:
  - <img width="931" alt="image" src="https://github.com/user-attachments/assets/478d73a0-4991-4cf1-ae1c-5a59ff4d455d">
- Click "Remove" for your own email. Instructions should look like:
  - <img width="813" alt="image" src="https://github.com/user-attachments/assets/00ca3e52-d780-4046-b6dc-7a916e7c1b5e">

## PR Checklist: Submitter


## PR Checklist: Reviewer


The larger the PR, the stricter we should be about these points.

## Pre Merge Checklist: Merger
